packages/wdio-cli/src/commands/config.ts

```diff
@@ -66,7 +67,8 @@ export const parseAnswers = async function (yes: boolean): Promise<ParsedAnswers
         presetPackage.package,
         ...reporterPackages.map(reporter => reporter.package),
         ...pluginPackages.map(plugin => plugin.package),
-        ...servicePackages.map(service => service.package)
+        ...servicePackages.map(service => service.package),
+        ...serenityPackages,
```

Comment:
I think so because `servicePackages` are determined based on what the developer selects in the configuration wizard, `serenityPackages` are separate from that concept. Is that what you meant?

Reply:
I see it now, I was on my phone and it was a bit hard to read and I read serenity twice but it's service and serenity.